lib/vrl/tests/tests/diagnostics/unnest_typedefs.vrl

```diff
@@ -1,26 +1,20 @@
 # result:
-# [true, true, true, true, true, true, true, true]
```

Comment:
The removed assertion here were asserting invalid coalescing behavior. There are now unit tests asserting the correct behavior.